documentation/modules/ROOT/pages/operations/openshift.adoc

```diff
@@ -8,56 +8,59 @@
+To get started even more quickly, try out the https://learn.openshift.com/middleware/debezium-getting-started/[{prodname} online learning scenario].
+It starts an OpenShift cluster just for you, which lets you start using {prodname} in your browser in just a few minutes.
 
 == {prodname} Deployment
 
-For setting up Kafka and Kafka Connect on OpenShift, a set of images provided by the https://strimzi.io/[Strimzi] project can be used, which offers "Kafka as a Service".
-It consists of enterprise grade configuration files and images that bring Kafka to OpenShift.
+To set up Kafka and Kafka Connect on OpenShift, use the set of images that are provided by the https://strimzi.io/[Strimzi] project. These images offer "Kafka as a Service" by providing enterprise grade configuration files and images that bring Kafka to OpenShift.
+
+.Prerequisites
+
+* Minishift is installed.
+* The OpenShift command line interface (`oc`) is installed.
 
-First, install the operators and templates for the Kafka broker and Kafka Connect into our OpenShift project:
+.Procedure
 
+. In your OpenShift project, enter the following commands to install the operators and templates for the Kafka broker and Kafka Connect:
++
 [source,shell,subs="attributes",options="nowrap"]
 ----
 export STRIMZI_VERSION={strimzi-version}
 git clone -b $STRIMZI_VERSION https://github.com/strimzi/strimzi-kafka-operator
 cd strimzi-kafka-operator
 
-# We need to create security objects as part of installation so it is necessary to switch to admin user
+# Switch to an admin user to create security objects as part of installation:
 oc login -u system:admin
 oc create -f install/cluster-operator && oc create -f examples/templates/cluster-operator
 ----
 
-Next, deploy a Kafka broker cluster and a Kafka Connect cluster and then create a Kafka Connect image with the {prodname} connectors installed:
-
+. Deploy a Kafka broker cluster and a Kafka Connect cluster and then create a Kafka Connect image with the {prodname} connectors installed:
++
 [source,shell,subs="attributes",options="nowrap"]
 ----
-# Deploy an ephemeral single instance Kafka broker
+# Deploy an ephemeral single instance Kafka broker:
 oc process strimzi-ephemeral -p CLUSTER_NAME=broker -p ZOOKEEPER_NODE_COUNT=1 -p KAFKA_NODE_COUNT=1 -p KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1 -p KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR=1 | oc apply -f -
 
-# Deploy a single instance of Kafka Connect with no plug-in installed
+# Deploy a single instance of Kafka Connect with no plug-in installed:
 oc process strimzi-connect-s2i -p CLUSTER_NAME=debezium -p KAFKA_CONNECT_BOOTSTRAP_SERVERS=broker-kafka-bootstrap:9092 -p KAFKA_CONNECT_CONFIG_STORAGE_REPLICATION_FACTOR=1 -p KAFKA_CONNECT_OFFSET_STORAGE_REPLICATION_FACTOR=1 -p KAFKA_CONNECT_STATUS_STORAGE_REPLICATION_FACTOR=1 -p KAFKA_CONNECT_VALUE_CONVERTER_SCHEMAS_ENABLE=false -p KAFKA_CONNECT_KEY_CONVERTER_SCHEMAS_ENABLE=false | oc apply -f -
 
 # Build a Debezium image
```

Comment:
Should this part of the procedure be replaced with a Dockerfile?
If yes, what exactly does the Dockerfile need to contain?

Reply:
Yes it should